pep-0001.txt

```diff
@@ -230,28 +238,40 @@ PEP Review & Resolution
 
 Once the authors have completed a PEP, they may request a review for
 style and consistency from the PEP editors.  However, the content and
-final acceptance of the PEP must be requested of the BDFL, usually via
-an email to the python-dev mailing list.  PEPs are reviewed by the
-BDFL and his chosen consultants, who may accept or reject a PEP or
-send it back to the author(s) for revision.  For a PEP that is
-predetermined to be acceptable (e.g., it is an obvious win as-is
-and/or its implementation has already been checked in) the BDFL may
-also initiate a PEP review, first notifying the PEP author(s) and
+final acceptance of the PEP must be requested of the core developers, usually
+via an email to the python-dev mailing list.  PEPs are reviewed by the
+core developers, who may accept or reject a PEP or send it back to the
+author(s) for revision.
+
+For a PEP that is predetermined to be acceptable (e.g., it is an obvious win
+as-is and/or its implementation has already been checked in) the Steering
+Council may also initiate a PEP review, first notifying the PEP author(s) and
```

Comment:
Hmm, why only for a predetermined PEP?  And if it is predetermined, then what is the purpose of the SC review?

Reply:
I was wondering why this was showing as a change, and then realised it was because I broke it up into two paragraphs.

I'll reword this to just say that the SC can initiate reviews, without the editorialising on why they might want to do that.

Reply:
And in attempting to do that, I realised why the edititorialising was there: without it, the sentence left me wondering why the SC would ever do that, given that the cost of leaving Open PEPs languishing is pretty low.

And the main reason for the SC to do this is because they want to *accept* the PEP, so the implementation can be merged before a release. Anyway, new wording coming in the next push.